CPAC/pipeline/engine.py

```diff
@@ -2403,7 +2454,7 @@ def strip_template(data_label, dir_path, filename):
     return data_label, json
 
 
-def ingress_pipeconfig_paths(cfg, rpool, unique_id, creds_path=None):
+def ingress_pipeconfig_paths(wf, cfg, rpool, unique_id, creds_path=None):
```

Comment:
Like this, this signature parameter prepending breaks a couple calls (https://github.com/FCP-INDI/C-PAC/blob/1a5ba5c40d119ee18248a51cfa17a3fb59afed98/CPAC/pipeline/test/test_engine.py#L93 and https://github.com/FCP-INDI/C-PAC/blob/1a5ba5c40d119ee18248a51cfa17a3fb59afed98/CPAC/resources/tests/test_templates.py#L35-L37) and should get a line in the CHANGELOG.